app/models/user.rb

```diff
@@ -48,6 +48,8 @@ class User < ApplicationRecord
   has_many :spots, dependent: :destroy
   has_many :first_matches, class_name: 'Match', foreign_key: 'first_user_id', dependent: :destroy
   has_many :second_matches, class_name: 'Match', foreign_key: 'second_user_id', dependent: :destroy
+  has_many :user_conversations
```

Comment:
Same as above

Reply:
I found that `dependent: :destroy` goes with `has_many through:` and it automatically removes the record at the join table when deleting it, maybe it´s the same: https://makandracards.com/makandra/32175-don-t-forget-automatically-remove-join-records-on-has_many-through-associations